src/coreclr-debug/debugConfigurationProvider.ts

```diff
@@ -59,6 +61,42 @@ export class DotnetDebugConfigurationProvider implements vscode.DebugConfigurati
                 return undefined;
             }
         }
+        
+        if (!this.platformInformation.isLinux() && !vscode.env.remoteName && vscode.env.uiKind != vscode.UIKind.Web) 
```

Comment:
>if [](http://example.com/codeflow?start=8&length=2)

Probably worth a comment explaining the reason for some of these checks. Something like:

// We want to ask the user if we should run `dotnet  dev-certs https --trust`, but this doesn't work in a few cases --
// Linux -- not supported by the .NET CLI as there isn't a single root cert store
// VS Code remoting/Web UI -- the trusted cert work would need to happen on the client machine, but we don't have a way to run code there currently
// `pipeTransport` -- the dev cert on the server will be different from the client